tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +348,177 @@ def test_del_obj_tag_40436(self):
         assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
                                                     f"Expected: {put_tag} \n Actual: {get_tag}")
         LOGGER.info("ENDED: Test DELETE object tagging in a versioning enabled bucket.")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.tags("TEST-40437")
+    @CTFailOn(error_handler)
+    def test_del_obj_tag_40437(self):
+        """
+        Test DELETE object tagging in a versioning suspended  bucket
+        """
+        LOGGER.info("STARTED: Test DELETE object tagging in a versioning suspended  bucket")
+
+        LOGGER.info("Step 1: PUT Bucket versioning with status as Enabled")
+        res = self.s3_ver_obj.put_bucket_versioning(bucket_name=self.bucket_name)
+        assert_utils.assert_true(res[0], res[1])
+
+        s3_ver_tlib.upload_version(self.s3_test_obj, bucket_name=self.bucket_name,
+                                   file_path=self.file_path1, object_name=self.object_name,
+                                   versions_dict=self.versions,
+                                   chk_null_version=True)
```

Comment:
```suggestion
                                   versions_dict=self.versions)
```

Reply:
This call will fail in validation as versioing is enabled for bucket in previous step.

Reply:
Removed